### PR TITLE
[Validator] Add missing validator polish translation

### LINKED ISSUE #43449: Missing translations for Polish (pl)
Hello,

There are some missing translation for Polish (pl). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by chikaldirick**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #43449
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf [hunk 1]
```diff
@@ -390,6 +390,10 @@
                 <source>This value should be a valid expression.</source>
                 <target>Ta wartość powinna być prawidłowym wyrażeniem.</target>
             </trans-unit>
+            <trans-unit id="101">
+                <source>This value is not a valid CSS color.</source>
+                <target>Ta wartość nie jest prawidłową CSS kolor.</target>
```
Comment (hywak) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: This translation isn't grammatically correct. It should be something like: `Ta wartość nie jest prawidłowym/poprawnym kolorem CSS`

**Review by hywak — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf (hunk 1 above)
Comment (IonBazan) [inline src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]:
```suggestion
                <target>Ta wartość nie jest prawidłowym kolorem CSS.</target>
```

@hywak Let's stick to "prawidłowym" as it's used in other messages too.

EDIT: Not sure how this initial message was translated but Google Translate does pretty good job:
image

**Review by IonBazan — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf (hunk 1 above)
Comment (hywak) [reply src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: No idea either 🤷🏼  `prawidłowym` sounds good. Thank you.

Comment (chikaldirick) [reply src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf]: Thanks for the suggestion, updated message according to it!

**Comment by fabpot** [comment]
Thank you @chikaldirick.

**Pull request merged**